dev/src/bundle.ts

```diff
@@ -19,14 +19,15 @@ import {firestore, google} from '../protos/firestore_v1_proto_api';
 import {DocumentSnapshot} from './document';
 import {QuerySnapshot} from './reference';
 import {Timestamp} from './timestamp';
+import {validateOptional, validateString} from './validate';
 
 import api = google.firestore.v1;
 
 /**
  * Builds a Firestore data bundle with results from the given queries and specified
  * documents.
  *
- * For documents in scope for multiple queries, only the latest version will
+ * For documents that are included multiple times, only the latest version will
```

Comment:
Independent of how we decide to implement this, should we relax this documentation to allow us to send multiple document version if the are processed "oldest to newest" first?